### PR TITLE
Added new language ST

### PR DESCRIPTION
**Pull request by Serhioromano**
This is one of the 5 languages of IEC-61331 standard for PLC program development.  It is usually not used as individual files of code but used in examples in articles or blog posts.

### CODE REVIEW

**Comment by Serhioromano** [comment]
What should I do to solve travis checks.

File: src/languages/smalltalk.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function(hljs) {
     begin: '#' + hljs.UNDERSCORE_IDENT_RE
   };
   return {
-    aliases: ['st'],
```
Comment (Serhioromano) [inline src/languages/smalltalk.js]: This one was really important as it conflicts with ST language. I think that alias is secondary to primary language name if it match.

**Comment by gusbemacbe** [comment]
@Serhioromano, @meseta has fixed it, he must receive the approval of us.

**Comment by meseta** [comment]
@gusbemacbe I'm not sure what's going on. Is there anything I can do to help?

**Comment by Serhioromano** [comment]
@gusbemacbe so what should I do so my addition pass the tests? I spent like 12 hours on that. I checked all other PRs that passes the test and they are the same like mine. I have no idea why it does not go through.

PS: Conflicts are resolved

**Comment by gusbemacbe** [comment]
You have to keep the 9.12 as @l has suggested.

And remove `AUTHOR.ru.txt` because it has been discarded by @isagalaev and @ldct.

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "syntax"
   ],
   "homepage": "https://highlightjs.org/",
-  "version": "9.12.0",
+  "version": "9.13.0",
```
Comment (gusbemacbe) [inline package.json]: As @ldct has suggested, you have to keep 9.12.

File: docs/conf.py [hunk 3]
```diff
@@ -48,9 +48,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '9.12'
+version = '9.13'
 # The full version, including alpha/beta/rc tags.
-release = '9.12.0'
+release = '9.13.0'
```
Comment (gusbemacbe) [inline docs/conf.py]: You have to keep 9.12 as @ldct  has suggested.

File: AUTHORS.ru.txt [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+- Джорди Петит <jordi.petit@gmail.com>
+- Рафаэль Паре <rparree@edc4it.com>
+- Джоель Порке <joel@porquet.org>
+- Алекс Арслан <ararslan@comcast.net>
+- Станислав Белов <stbelov@gmail.com>
+- Иван Дементьев <ivan_div@mail.ru>
+- Николя Ллобера <nllobera@gmail.com>
+- Мортен Пибелехт <morten.piibeleht@gmail.com>
+- Мартин Клаузен <martin.clausene@gmail.com>
+- Сергей Романов <serg4172@mail.ru>
```
Comment (gusbemacbe) [inline AUTHORS.ru.txt]: You have to remove it because it has been discarded by @isagalaev and @ldct.

**Comment by gusbemacbe** [comment]
Please, read the rules of @ldct: https://github.com/highlightjs/highlight.js/pull/1724#pullrequestreview-139299268

You should not commit `package.json`.

File: AUTHORS.ru.txt (hunk 4 above)
Comment (Serhioromano) [reply AUTHORS.ru.txt]: I am not sure how to delete this. This happened after my conflict resolve. Before it was only one line. But I am sure this is not the reason tests do not pass.

How to delete this file from commit?

Comment (gusbemacbe) [reply AUTHORS.ru.txt]: Your last branch you merged and pushed, fixing the conflicts... The latest branch was yesterday. Make backup of the ST language JavaScript file and fix again merge conflicts of the latest version/branch merged and pulled by @ldct yesterday. His latest version will delete automatically the unnecessary files.

**Comment by Serhioromano** [comment]
I think I'll just make another PR. You can decline this one. BИусфгыу  cannot understand even with your explanation how to fix this AUTHOR file now adds all strings as new.

**Comment by gusbemacbe** [comment]
@Serhioromano, BИусфгыу? What is it?

**Comment by gusbemacbe** [comment]
I understood now. You wrote "because" in Russian keyboard.

**Comment by gusbemacbe** [comment]
@ldct, is there this to decline this PR that he will create a new PR?

**Comment by ldct** [comment]
closing as requested by @gusbemacbe 

@Serhioromano thanks, and looking forward to a separate PR without the merge conflicts!

**Pull request closed without merging**